postgres/osmap.yaml

```diff
@@ -91,9 +91,45 @@ RedHat:
 {% endif %}
 
 Suse:
+  pkg_repo:
+    name: pgdg-sles-{{ release }}
+    humanname: PostgreSQL {{ repo.version }} $releasever - $basearch
+    #Using sles-12 upstream repo for opensuse
+    baseurl: 'https://download.postgresql.org/pub/repos/zypp/{{ repo.version }}/suse/sles-12-$basearch'
+    key_url: 'https://download.postgresql.org/pub/repos/zypp/{{ repo.version }}/suse/sles-12-$basearch/repodata/repomd.xml.key'
+    gpgcheck: 1
```

Comment:
Just add
```yaml
    gpgautoimport: True
```
 
bellow, and importing GPG key from the upstream will start to magically work. No need for additional `cmd.run` state in the `upstream.sls`.

Reply:
Thanks - done - let me know when your PR comments are done and I'll do some retesting on Suse to make sure there is no regression.